wiki/Help_Centre/Beatmapping/en.md

```diff
@@ -0,0 +1,67 @@
+# Beatmapping (help page)
+
+*For a list of all Help Centre articles, see: [Help Centre](/wiki/Help_Centre)*
+
+## How do I upload/update my map?
+
+**You can do this through the in-game editor.**
+
+To upload your beatmap, go into the editor and hit `File` > `Upload Beatmap`.
+
+This will make your beatmap available for others to download on the website and create a dedicated thread for discussion. Updating a map is the same process for a map that has already been uploaded.
+
+Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (by not updating it or not posting in its thread) will send it to the Beatmap Graveyard. To update a graveyarded beatmap, you'll need to revive it first.
+
+## The editor is telling me that the beatmap I'm trying to submit isn't mine!
+
+**This can happen if the creator of the beatmap is not properly set within the beatmap files.**
+
+In the editor, click `File` in the menu, then `Open .osu file in Notepad`. Check the line `Creator:` and make sure that it's set to your exact username, capitalization included.
+
+If it isn't, manually change it for every difficulty in your set, reload the map, and you should be able to submit it.
+
+### I'm an osu!supporter, but I don't have 10 pending slots!
+
+**The number of slots available depends on the number of Ranked maps you have.**
+
+By default, every user has **2 pending slots**. 1 slot is added for every Ranked map the user owns, up to a total of 8 slots. Being a supporter grants 2 additional slots, from 4 (without Ranked maps), up to a total of 10.
```

Comment:
I think these numbers might be a bit outdated, probably a good idea to change them now. The numbers under the "Upload More Beatmaps" section of https://osu.ppy.sh/home/support should be correct.